vertx-sql-client-template/src/main/asciidoc/index.adoc

```diff
@@ -0,0 +1,325 @@
+= Client SQL templates
+:toc:
+
+SQL client templates is a small library design to facilitate the execution of SQL queries.
+
+== Usage
+
+To use client SQL templates add the following dependency to the _dependencies_ section of your build descriptor:
+
+* Maven (in your `pom.xml`):
+
+[source,xml]
+----
+<dependency>
+  <groupId>${maven.groupId}</groupId>
+  <artifactId>${maven.artifactId}</artifactId>
+  <version>${maven.version}</version>
+</dependency>
+----
+* Gradle (in your `build.gradle` file):
+
+[source,groovy]
+----
+dependencies {
+  compile '${maven.groupId}:${maven.artifactId}:${maven.version}'
```

Comment:
Use `implementation` rather than `compile`.

Reply:
But that's a more general remark. `compile` is still here, but it's recommended to go with `implementation` for dependencies you consume, and `api` for dependencies you want to consume the re-export.